tests/test_vqe.py

```diff
@@ -79,17 +79,17 @@ def seed():
 ]
 
 valid_hamiltonians_str = [
-    "(1.0) [Hermitian0'1]",
-    "(-0.8) [Z0]",
-    "(0.6) [X0 X1]",
-    "(0.5) [X0]\n+ (-1.6) [Y1]",
-    "(0.5) [X1]\n+ (-1.6) [Y1]",
-    "(0.5) [Xa]\n+ (-1.6) [Yb]",
-    "(1.1) [X0]\n+ (-0.4) [Hermitian2]\n+ (0.333) [Z2]",
-    "(-0.4) [Hermitian0'2]\n+ (0.15) [Z1]",
-    "(1.5) [Z0]\n+ (2.0) [Y2]",
-    "(-0.1) [Hermitian0'1]\n+ (0.5) [Y0]",
-    "(0.5) [X0]\n+ (1.2) [X0 X1]",
+    "  (1.0) [Hermitian0'1]",
```

Comment:
Could changing the spacing have any side effects? Maybe demos outputs/existing docstrings might not match any more?

Reply:
Demo's won't match up anymore, but also because the terms will be in different orders.

docstrings might be an issue.  I've been thinking as well about going through docstrings and making sure code samples still work.  I think there may be multiple places code examples need to get updated.  I made an Issue the other day about circuit drawing in the permute template.

Reply:
I saw, that was a good catch!